moped-editor/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "atd-moped-editor",
   "author": "ATD Data & Technology Services",
   "license": "CC0 1.0 Universal",
-  "version": "1.0.2",
+  "version": "1.1.0",
```

Comment:
I have an open PR to do this same change, which you have approved. It's still waiting on someone from Product.

I think it's totally cool to put it in this PR as well, and if this goes in first, which is likely, I'll close out the other PR since it will have become a non-op. 

Thanks for bumping this in `main`!

Reply:
Yeah I saw that and figured I'd sneak it in here. We can close out your PR